tests/conftest.py

```diff
@@ -2,19 +2,15 @@
 
 import pytest
 from _pytest.fixtures import FixtureRequest
-from src.web3_extentions.typings import Web3
+from web3 import Web3
 from web3.providers import JSONBaseProvider
 
-from src import variables
+from src.variables import CONSENSUS_CLIENT_URI, EXECUTION_CLIENT_URI
 from src.typings import SlotNumber
+from src.web3_extentions import LidoContracts, LidoValidatorsProvider, TransactionUtils, KeysAPIClientModule
 from tests.mocks import chain_id_mainnet, eth_call_el_rewards_vault, eth_call_beacon_spec
-from tests.providers import (
-    ResponseToFileProvider,
-    ResponseFromFile,
-    MockProvider,
-    ResponseToFileHTTPClient,
-    ResponseFromFileHTTPClient,
-)
+from tests.providers import ResponseToFileProvider, ResponseFromFile, MockProvider, ResponseToFileConsensusClientModule, \
```

Comment:
Let's use https://peps.python.org/pep-0328/#rationale-for-parentheses to group split imports in few lines